pkgs/development/tools/analysis/retdec/default.nix

```diff
@@ -139,10 +151,16 @@ stdenv.mkDerivation (self: {
       url = "https://github.com/avast/retdec/commit/dbaab2c3d17b1eae22c581e8ab6bfefadf4ef6ae.patch";
       hash = "sha256-YqHYPGAGWT4x6C+CpsOSsOIZ+NPM2FBQtGQFs74OUIQ=";
     })
+    # aarch64 compatibility: https://github.com/avast/retdec/pull/1195
+    (fetchpatch {
+      url = "https://github.com/avast/retdec/commit/93233e79a0b14e6aeeee6b117a1c93feba88577b.patch";
+      hash = "sha256-0VT7J9mcrneUo80BSi+unqCGdNNp4HaRx615M6hm0n8=";
+    })
```

Comment:
Since this is unmerged, we’d prefer to have a vendored patch in‐tree to avoid GitHub GC issues.